receiver/journaldreceiver/coverage.out

```diff
@@ -0,0 +1 @@
+mode: atomic
```

Comment:
I think this file is supposed to be excluded?

Reply:
Yes, thank you for pointing this. Removed it